Sources/Confidence/ConfidenceClient/RemoteConfidenceClient.swift

```diff
@@ -0,0 +1,112 @@
+import Foundation
+import Common
+import os
+
+public class RemoteConfidenceClient: ConfidenceClient {
+    private var options: ConfidenceClientOptions
+    private let metadata: ConfidenceMetadata
+    private var httpClient: HttpClient
+    private var baseUrl: String
+
+    init(
+        options: ConfidenceClientOptions,
+        session: URLSession? = nil,
+        metadata: ConfidenceMetadata
+    ) {
+        self.options = options
+        switch options.region {
+        case .global:
+            self.baseUrl = "https://events.confidence.dev/v1/events"
+        case .europe:
+            self.baseUrl = "https://events.eu.confidence.dev/v1/events"
+        case .usa:
+            self.baseUrl = "https://events.us.confidence.dev/v1/events"
```

Comment:
We will probably make the RemoteConfidenceClient more generic later on and capable to handle both event uploads and flag resolves. But keeping this for a future PR